packages/perseus-core/src/utils/util.ts

```diff
@@ -0,0 +1,688 @@
+import _ from "underscore";
+
+import type {Range} from "../data-schema";
+
+type WordPosition = {
+    start: number;
+    end: number;
+};
+
+type WordAndPosition = {
+    string: string;
+    pos: WordPosition;
+};
+
+type RNG = () => number;
+
+export type ParsedValue = {
+    value: number;
+    exact: boolean;
+};
+
+// TODO: dedupe this with Coord in interactive2/types.js
+type Coordinates = [number, number];
+
+export type GridDimensions = {
+    scale: number;
+    tickStep: number;
+    unityLabel: boolean;
+};
+
+type QueryParams = {
+    [param: string]: string;
+};
+
+export type Position = {
+    top: number;
+    left: number;
+};
+
+type TouchHandlers = {
```

Comment:
Having references to UI concepts in `perseus-score` raises a warning flag for me.  What do you think of leaving anything to do with UI (touch, event handling, anything touching `window`).